packages/lit-dev-content/site/_includes/default.html

```diff
@@ -40,7 +40,7 @@
 
     <link rel="preconnect" href="https://fonts.gstatic.com">
 
-    <link href="/blog/atom.xml" rel="alternate" type="application/atom+xml" data-title="Lit Blog feed">
+    <link href="/blog/atom.xml" rel="alternate" type="application/atom+xml" title="Lit Blog feed">
```

Comment:
Is there a pro/con to having both attributes supported? Non blocking.

Reply:
The data-title attribute does nothing as far as I know, but title does something